modules/oe_whitelabel_modal_page/tests/src/Functional/ModalFormFieldAccessTest.php

```diff
@@ -43,20 +69,23 @@ public function testAdvancedFieldsHiddenWithoutPermission(): void {
     $assert_session->fieldExists('Title');
     $assert_session->fieldExists('Pages');
 
-    // Assert that advanced fields are NOT accessible.
-    $assert_session->fieldNotExists('Open this modal clicking on this element');
-    $assert_session->fieldNotExists('Auto Open');
-    $assert_session->fieldNotExists('Prevent Default');
-    $assert_session->fieldNotExists('Class(es)', $assert_session->elementExists('xpath', '//details[./summary[.="MODAL HEADER"]]'));
-    $assert_session->fieldNotExists('Class(es)', $assert_session->elementExists('xpath', '//details[./summary[.="MODAL FOOTER"]]'));
-    $assert_session->elementNotExists('xpath', '//details[./summary[.="Cookies"]]');
-    $assert_session->fieldNotExists('Label', $assert_session->elementExists('xpath', '//details[./summary[.="Button X close"]]'));
-    $assert_session->fieldNotExists('ok_button_class');
-    $assert_session->fieldNotExists('left_button_class');
-    $assert_session->elementNotExists('xpath', '//details[./summary[.="Maximize Button"]]');
-    $assert_session->elementNotExists('xpath', '//details[./summary[.="MODAL CLASS"]]');
-    $assert_session->fieldNotExists('Parameters');
-    $assert_session->fieldNotExists('Modal By');
+    $field_assert = $should_exist ? 'fieldExists' : 'fieldNotExists';
+    $element_assert = $should_exist ? 'elementExists' : 'elementNotExists';
```

Comment:
Alternative would be anonymous / arrow functions.
```suggestion
    $field_assert = $should_exist ? $assert_session->fieldExists(...) : $assert_session->fieldNotExists(...);
    $element_assert = $should_exist ? $assert_session->elementExists(...) : $assert_session->ementNotExists(...);
```
and
```php
    $field_assert('Open this modal clicking on this element');
```
(actually the var names would be different, e.g. `$assert_field_if`.)

The question is whether this would make the code below more readable.
Clearly it would be shorter, but the current version makes it clear that these are assert methods.

So ok for me to leave it as is.